.github/workflows/forms-flow-documents-cd.yml

```diff
@@ -1,93 +1,165 @@
-name: Push Forms flow Document to registry
+name: forms-flow-documents-deploy
 
 on:
   workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+        - dev
+        - test
+        - production
   push:
-    branches: [ master, develop ]
+    branches: [main, feature/DGJ_666-add-aot-pdf-generator]
```

Comment:
We can probably remove the feature branch from here now :)

Reply:
addressed